src/components/Text.js

```diff
@@ -24,6 +24,11 @@ const Text = ({
   lineHeight = 1.5,
   underline,
   display = 'inline-block',
+  wordBreak = 'normal',
+  overflowWrap = 'normal',
+  WebkitHyphens = 'none',
+  msHyphens = 'none',
+  hyphens = 'none',
```

Comment:
I think we should have fewer props, can we simplify this? We could start with leaving the vendor prefix version complications at this level, and providing a simpler API to the consumers of `Text`